### DISCUSSION TITLE
Multi files renamer

### DISCUSSION THREAD

**Discussion by lorens-osman-dev** [spacedriveapp/spacedrive#2323] [Feature Request]
Thunar is One of Linux's best file managers, It has a "Bulk file rename" plugin which change multi files names at once with ton of features, it is great useful tool, I think Spacedrive must have such like unique tool.
[More Information ](https://www.maketecheasier.com/bulk-rename-files-linux-thunar-bulk-rename-tool/)

**Comment by ketonik** [comment]
Probably outside the scope of a file manager I rekon, cos we'll have people wanting more functionality from it which ends up being it's own dedicated app like Name Mangler, which incidentally is a brilliant app on macOS, shame nothing like it on dumDOZE.

https://manytricks.com/namemangler/
